api/Metrics/UpDownCounter.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Metrics;
+
+interface UpDownCounter extends Counter
+{
```

Comment:
Should we add `@access` and `@return` doc blocks here?

Reply:
done